content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -70,6 +70,9 @@ An _output entry_ (`outs`) consists of these fields:
   [ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation) for
   HTTP, S3, or Azure [external outputs](/doc/user-guide/managing-external-data);
   and a special _checksum_ for HDFS.
+- `size` (optional): Size of the file or directory (sum of all files in the
+  directory).
+- `nfiles` (optional): Number of files in the directory.
```

Comment:
No need for "optional" because most fields are optional so we don't use that except for `meta` (we have "always present" in a few instead).

```suggestion
- `size`: Size of the file or directory (sum of all files).
- `nfiles`: If a directory, number of files inside.
```